packages/react-compass/src/tabs/tab/index.styles.ts

```diff
@@ -75,6 +75,29 @@ export const StyledTab = styled('div', {
           },
         },
       },
+      h5: {
+        background: 'transparent',
+        padding: '0',
+        color: '$grayShades60',
+        fontSize: '$label1',
+        lineHeight: '20px',
```

Comment:
use variable in `lineHeights` object which is defined in `theme.ts`

Reply:
@pieliedie  There is no variable with a line height of 20px